Cargo.toml

```diff
@@ -26,7 +26,7 @@ parley = { version = "0.4.0", default-features = false, path = "parley" }
 peniko = { version = "0.4.0", default-features = false }
 skrifa = { version = "0.31.0", default-features = false }
 read-fonts = { version = "0.29.0", default-features = false }
-swash = { version = "0.2.4", default-features = false }
+swash = { git = "https://github.com/dfrg/swash", rev = "fbc4c5aea5167fa10b92b453d9e308934e7232fd", default-features = false }
```

Comment:
We should probably avoid depending on a git version of Swash if possible. If you put up a PR bumping the version then I can publish a patch release of Swash with the required Debug impl.

Reply:
Sure, opened [swash#100](https://github.com/dfrg/swash/pull/100). Merge it and publish.

Reply:
Swash v0.2.5 published :)